pom.xml

```diff
@@ -165,6 +165,26 @@
             <version>2.2</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.7.26</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jcl-over-slf4j</artifactId>
+            <version>1.7.26</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>log4j-over-slf4j</artifactId>
+            <version>1.7.26</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
```

Comment:
I'm confused why the sudden addition of `slf4j` dependencies, and none of them are from dependency management. 
Could you elaborate?

Reply:
Upper bound dependency issue. Looks like bumping the parent pom version solves it, so I'll do that instead (it creates the issue for hamcrest though)